### PR TITLE
Feature v1.0 crm1114

### PR DESCRIPTION
**Pull request by SlalomAK**

### CODE REVIEW

File: src/classes/TestOpportunityPlaybookController.cls [hunk 1]
```diff
@@ -354,4 +354,137 @@ private class TestOpportunityPlaybookController {
             test.stopTest();
          }
     }
-}
\ No newline at end of file
+
+    static testMethod void testInsertPrivateExchangeBlueDirectionsCase(){
+        Id caseRecordTypeIdPEBD = Schema.SObjectType.Case.getRecordTypeInfosByName().get('Private Exchange - Blue Directions').getRecordTypeId();
+        Id accountRecordTypeGroup = Schema.SObjectType.Account.getRecordTypeInfosByName().get('Group Account').getRecordTypeId();
```
Comment (rbonslater) [inline src/classes/TestOpportunityPlaybookController.cls]: @SlalomAK we could use a preexisting custom label here (Label.Group_Account)

File: src/classes/OpportunityPlaybookController.cls [hunk 2]
```diff
@@ -619,6 +619,24 @@ public with sharing class OpportunityPlaybookController
...
+                if(tempAccount.NumberOfEmployees != null)
+                    caseRec.Group_Size_e__c = tempAccount.NumberOfEmployees;
+                if(tempAccount.Group_Associated__r.size() > 0)
+                    if(tempAccount.Group_Associated__r.get(0).Producer_Account__c != null)
+                        caseRec.Consultant_Broker__c = tempAccount.Group_Associated__r.get(0).Producer_Account__c; //Most recent
+                for(AccountTeamMember member: tempAccount.AccountTeamMembers){
+                    if(member.TeamMemberRole == 'Account Executive' && caseRec.Account_Executive__c == null){
+                        caseRec.Account_Executive__c = member.UserId;
+                    }
+                    else if(member.TeamMemberRole == 'Underwriter' && caseRec.Name_Of_Current_Medical_Underwriter__c == null){
```
Comment (rbonslater) [inline src/classes/OpportunityPlaybookController.cls]: @SlalomAK could we use a custom label for 'Underwriter'? Label.AccountUnderwriterRole

**Comment by SlalomAK** [comment]
@rbonslater fixed

**Comment by rbonslater** [comment]
@SlalomAK looks good. 
@fatimasana

**Pull request merged**